src/Detail/Importing/Source/DBAL/MssqlSource.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+namespace Detail\Importing\Source\DBAL;
+
+use Doctrine\DBAL\Driver as DBALDriver;
+
+use Detail\Importing\Source\DBALSource;
+
+abstract class MssqlSource extends DBALSource
```

Comment:
Am I right in my assumption that we don't need this class when we rely on `Doctrine\DBAL\Query\QueryBuilder` for building queries (see comment below)?